assignment/assignment-tool/tool/src/java/org/sakaiproject/assignment/tool/AssignmentAction.java

```diff
@@ -1675,6 +1685,17 @@ protected String build_student_view_submission_context(VelocityPortlet portlet,
 				context.put("student",student);
 			}
 		}
+
+		try {
+			Site st = SiteService.getSite((String) state.getAttribute(STATE_CONTEXT_STRING));
+			context.put("isAdditionalNotesEnabled", candidateDetailProvider.isAdditionalNotesEnabled(st));
```

Comment:
Isn't this going to throw a NPE when you are not running in the demo setup and don't have a `candidateDetailProvider` configured?